### ISSUE TITLE
Reagarding discripency in tensorboard validation accuracy and the one printed during training

### ISSUE THREAD

**Issue by ankit1-iitbhu** [tensorflow/tensorflow#46943]
When I am running a random search for hyperparameter tunning. The accuracies appearing on print screen for training and validation sets are different which is not appearing in tensorboard. In tensorboard the validation accuracy always appears lesser than the print value.

![image](https://user-images.githubusercontent.com/37747800/106998754-040d9680-67ab-11eb-8833-0ed9b6b7bf4c.png)
![image](https://user-images.githubusercontent.com/37747800/106998902-40d98d80-67ab-11eb-9713-f9e4efba3a5a.png)

The accuracy appearing are different. Why?

**Comment by amahendrakar** [comment]
@ankit1-iitbhu,
Tensorboard issues are tracked in tensorflow/tensorboard repo. Could you please submit a new issue from [this link](https://github.com/tensorflow/tensorboard/issues/new/choose) and fill in the template, so that we can track the issue there. Thanks!
